wp-content/themes/core/blocks/core/lists/style.pcss

```diff
@@ -4,6 +4,12 @@
 
 /* ----------------------------------------------------------------------
  * Ordered List
+ *
+ * The first selector applies to the FE, the second in the editor.
+ * The list block is the only block on the FE that doesn't get a class,
+ * so we're able to apply a :not() selector for the class attribute.
+ * While both will apply in the editor, only the second will affect
+ * the actual list block.
  * ---------------------------------------------------------------------- */
 
 .wp-site-blocks ol:not([class]),
```

Comment:
❓  Just so I'm clear, the selector `.wp-site-blocks` IS rendered on the front-end somewhere? Is that a class we're adding to the templates somewhere? I'm not familiar with that class in the output of a public site.

Reply:
Correct yeah, `wp-site-blocks` is the wrapping class for all blocks on the FE & editor.